app/controllers/AssessorController.scala

```diff
@@ -42,7 +42,7 @@ trait AssessorController extends BaseController {
   }
 
   def addAvailability(userId: String): Action[JsValue] = Action.async(parse.json) { implicit request =>
-    withJsonBody[AssessorAvailability] { availability =>
+    withJsonBody[AssessorAvailabilityOld] { availability =>
```

Comment:
Yes totally agree, I will remove it in my current story. I wanted to implement the storage of availability so that Benji can have a look but there were more things to change than expected. My fault.